crates/solidity/inputs/language/src/dsl.rs

```diff
@@ -580,7 +580,7 @@ slang_grammar! {
 
     } ;
 
-    lexical context VersionPragma = {
+    lexical context Pragma = {
```

Comment:
is there any remaining usage of this file? should we delete it in this PR as well?

Reply:
This change is a leftover when I was trying to see where and how much do the v1 and v2 differ.

I felt more comfortable doing this separately to limit the scope of the PR, that's the reason why this doesn't include revamped codegen grammar traits/interface, so that we can support both for the brief period of transition.

Do you want me to remove this file as part of this PR?

Reply:
I suggest so. Since we are no longer using it, removing it means that we no longer have to keep it up to date (one down, one to go 🎉).

Reply:
Removed in 52f522bc6a2276b770f0ec66a23169d09c4a5253